package/cluster/network/composition.yaml

```diff
@@ -30,39 +94,55 @@ spec:
               Name: platformref-vpc
       name: platformref-vcp
       patches:
+        - type: PatchSet
+          patchSetName: region
+        - type: PatchSet
+          patchSetName: deletionPolicy
         - type: PatchSet
           patchSetName: network-id
     - base:
         apiVersion: ec2.aws.upbound.io/v1beta1
         kind: InternetGateway
         spec:
           forProvider:
-            region: us-west-2
+            # region: us-west-2
             vpcIdSelector:
               matchControllerRef: true
       name: gateway
       patches:
+        - type: PatchSet
+          patchSetName: region
+        - type: PatchSet
+          patchSetName: deletionPolicy
         - type: PatchSet
           patchSetName: network-id
     - base:
         apiVersion: ec2.aws.upbound.io/v1beta1
         kind: Subnet
         metadata:
           labels:
-            zone: us-west-2a
+            zone: us-region-zone-a
             access: public
         spec:
           forProvider:
-            region: us-west-2
+            # region: us-west-2
```

Comment:
Comments can be removed now that the patching is working.

Reply:
Removed comments.